### PR TITLE
[RunAllTests] Fix #3837 tests end in "Test"

### LINKED ISSUE #3837: Make sure all tests end in "Test"
Currently, at least two tests aren't ending with "Test": ProfileProgressSpanCount & PlayerSplitScreenTesting. Both should be renamed such that they end in "Test", and all other files in test folders should be double checked to make sure that they end with "Test".

### PR DESCRIPTION
**Pull request by rishidyno**
Fix #3837
## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/test/java/org/oppia/android/app/testing/player/split/PlayerSplitScreenTest.kt
Comment (yash10019coder) [inline app/src/test/java/org/oppia/android/app/testing/player/split/PlayerSplitScreenTest.kt:207]: Rename at line 177 DaggerPlayerSplitScreenTest_TestApplicationComponent.builder()

**Review by yash10019coder — COMMENTED** [review]
Hi @rishidyno look at the comment thanks

**Comment by rishidyno** [comment]
@rt4914 All tests have passed PTAL. Also I don't think we have any other files whose name needs to be changed.

File: scripts/assets/test_file_exemptions.textproto
Comment (rt4914) [inline scripts/assets/test_file_exemptions.textproto]: I think these two lines can be removed now. Can you try this change and see if CI passes?

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rishidyno Thanks.

File: scripts/assets/test_file_exemptions.textproto
Comment (rishidyno) [reply scripts/assets/test_file_exemptions.textproto]: @rt4914  All tests passed. PTAL.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks

**Review by BenHenning — COMMENTED** [review]
This is a nice cleanup, thanks @rishidyno! Apologies for the late review.

I think the PR is correct, but for some reason the affected test computation isn't including the new tests. I'm not sure if this is an issue with the build graph (since these weren't picked up before due to not ending in 'Test', hence the need for this change), or with the script. I need to investigate before approving since we need to make sure that these tests now pass on Bazel in CI before merging.

**Comment by BenHenning** [comment]
Odd. I see from CI the following affect test computation:

``Affected Bazel test targets: [//scripts/src/javatests/org/oppia/android/scripts/testfile:TestFileCheckTest]``

But when run locally on the branch, I see:

```
[//app:src/test/java/org/oppia/android/app/testing/ProfileProgressSpanCountTest, //app:src/test/java/org/oppia/android/app/testing/player/split/PlayerSplitScreenTest, //scripts/src/javatests/org/oppia/android/scripts/testfile:TestFileCheckTest]
```

For some reason the moved files weren't considered changed by Git when the CI workflow was run. Going to try running it again to see if that changes anything.

**Comment by BenHenning** [comment]
Oddly enough, I can consistently get correct results locally & not in CI. Created #4034 to investigate further, and fortunately it repros in that PR. Unfortunately, this PR needs to block until we have a fix.

**Comment by BenHenning** [comment]
Aha! I think I figure out an important difference. When running locally, the common merge commit with develop is 10fa6cd5145f08217971787a795cd3b07965ebc0 (which is correct), whereas in CI it's 4a897a17ef35c8db54fa8548edc38d6779cc2870 (which is the latest develop change). Since the PR isn't up-to-date, the delta is computing incorrectly for CI. This is a problem and needs to be fixed.

**Comment by BenHenning** [comment]
#4035 was filed to track this.

**Review by anandwana001 — APPROVED** [review]
LGTM for codeowner files.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Requesting changes since this can't be merged yet (see my messages above).

**Comment by BenHenning** [comment]
@rishidyno I was actually investigating this issue (Oppiabot wasn't supposed to reassign it). I'm still looking into fixing the CI script, though syncing with develop will force the CI flow to run with the latest changes.

Edit: apparently not. The latest workflow is still missing the new tests & the merge-base hasn't changed. Hmm.

**Comment by rishidyno** [comment]
> @rishidyno I was actually investigating this issue (Oppiabot wasn't supposed to reassign it). I'm still looking into fixing the CI script, though syncing with develop will force the CI flow to run with the latest changes.
> 
> Edit: apparently not. The latest workflow is still missing the new tests & the merge-base hasn't changed. Hmm.

ooh! I see oppia bot is not working appropriately.  So new tests have to be written for the latest workflow then only this pr can be merged? As for now, I have updated this branch with develop as you mentioned above.

**Comment by BenHenning** [comment]
> > @rishidyno I was actually investigating this issue (Oppiabot wasn't supposed to reassign it). I'm still looking into fixing the CI script, though syncing with develop will force the CI flow to run with the latest changes.
> > Edit: apparently not. The latest workflow is still missing the new tests & the merge-base hasn't changed. Hmm.
> 
> ooh! I see oppia bot is not working appropriately. So new tests have to be written for the latest workflow then only this pr can be merged? As for now, I have updated this branch with develop as you mentioned above.

This PR can't be merged until all the tests are running in Bazel (at the moment, they aren't). I'm working on figuring out what's broken.

Also to be clear: Oppiabot is working, I just didn't intend for it to reassign the PR.

**Comment by rishidyno** [comment]
> > > @rishidyno I was actually investigating this issue (Oppiabot wasn't supposed to reassign it). I'm still looking into fixing the CI script, though syncing with develop will force the CI flow to run with the latest changes.
> > > Edit: apparently not. The latest workflow is still missing the new tests & the merge-base hasn't changed. Hmm.
> > 
> > 
> > ooh! I see oppia bot is not working appropriately. So new tests have to be written for the latest workflow then only this pr can be merged? As for now, I have updated this branch with develop as you mentioned above.
> 
> This PR can't be merged until all the tests are running in Bazel (at the moment, they aren't). I'm working on figuring out what's broken.
> 
> Also to be clear: Oppiabot is working, I just didn't intend for it to reassign the PR.

Oh! got it.

**Comment by BenHenning** [comment]
@rishidyno I was going to retrigger the CI workflow to unblock this PR so that it can get merged, but it seems that your fork was deleted which means the PR is now an orphan. Unfortunately, you'll need to open a new PR with the same changes.

**Comment by rishidyno** [comment]
@BenHenning Actually my old fork got messed up so I had to delete it. I have created new PR for the same https://github.com/oppia/oppia-android/pull/4062

**Pull request closed without merging**
